app.js

```diff
@@ -22,4 +24,29 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+  .then((beers) => {
+    res.render('beers', {beers});
+  })
+  .catch(e => res.status(500).send("Something went wrong"));
+})
+
+app.get('/beers/:id', (req, res) => {
+  punkAPI.getBeer(req.params.id)
+  .then((beer) => {
+    res.render('random-beer', {beer});
+  })
+  .catch(e => res.status(500).send("Something went wrong"));
+})
+
+app.get('/random-beers', (req, res) => {
+  punkAPI.getRandom()
+  .then((beer) => {
+    res.render('random-beer', {beer});
+  })
+  .catch(e => res.status(500).send("Something went wrong"));
+})
+
+
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
```

Comment:
Just wanted to mention that it's considered common practice to store the port in a variable `const port = 3000;` then you can refer to it `app.listen(port, () => console.log(`🏃‍ on port ${port}`))`